### DISCUSSION TITLE
Recommended specs for VMs running docker-valhalla

### DISCUSSION THREAD

**Discussion by isakengstrom** [gis-ops/docker-valhalla#90] [General]
Previously (up to 3.3.0) I've had the container running on a VM with 1 cpu core and 1.5GB of memory, but with the latest release, this does not seem to be enough. Attempts result in 3x startup time (around 1.5 hours), followed by a crash when the container is about to finish the startup. Increasing the VM's capacity to 2 cores and 3GB of memory seem to fix this issue. Are there any recommended minimum spec requirements?

**Comment by nilsnolde** [comment]
It’s not that easy to determine minimum specs for building tiles. Just gotta trial & error if you’re on modest hardware. 

Note that the runtime for the service requires much less memory.
